packages/vaadin-rich-text-editor/test/basic.test.js

```diff
@@ -45,10 +45,8 @@ describe('rich text editor', () => {
         await nextRender(rte);
       });
 
-      // FIXME: flaky tests in GitHub Actions only in Firefox (passing locally).
       ['bold', 'italic', 'underline', 'strike', 'blockquote', 'code-block'].forEach((fmt) => {
-        const isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
-        (isFirefox || isSafari ? it.skip : it)(`should apply ${fmt} formatting to the selected text on click`, () => {
+        it(`should apply ${fmt} formatting to the selected text on click`, () => {
```

Comment:
Reenabled the test for FF, let's see if it works in CI

Reply:
Build passed, let's see if there is some flakiness in future. If so, we can create separate investigation ticket.